services/QuillLMS/client/app/bundles/Teacher/components/dashboard/handy_actions.tsx

```diff
@@ -25,18 +26,24 @@ const ExploreDemoAction = ({setShowDemoModal}) => (
   </button>
 )
 
-const HandyActions = ({ linkedToClever, setShowDemoModal}) => (
-  <section className="handy-actions">
+const HandyActions = ({ linkedToClever, setShowDemoModal}) => {
+  const [errors, setErrors] = React.useState<string[]>([])
+  const [hasAppSetting, setHasAppSetting] = React.useState<boolean>(false);
+  React.useEffect(() => {
+    handleHasAppSetting({appSettingSetter: setHasAppSetting, errorSetter: setErrors, key: 'demo', })
+  });
```

Comment:
Nitpicky but you want this line to be `}, [])` - otherwise this will still run on every re-render.